src/drc/api/serializers.py

```diff
@@ -164,16 +188,15 @@ def validate_indicatie_gebruiksrecht(self, indicatie):
 
     def validate(self, attrs):
         valid_attrs = super().validate(attrs)
-
-        lock = valid_attrs.get('lock', '')
+        lock = self.context['request'].data.get('lock', '')
```

Comment:
you're now using unvalidated data in your processing, the `lock` field must be validated on presence/absence and format _before_ using it for further processing.

Probably best to define an explicit `lock` field on the serializer that doesn't (yet) map to the model, and it needs the write_only + help text configuration that was just removed. The `lock` field _must_ show up in the API spec, which it does if it's a serializer field.